package.json

```diff
@@ -11,8 +11,9 @@
     "test": "jest --rootDir=specs --verbose"
   },
   "dependencies": {
+    "@react-native-community/masked-view": "^0.1.10",
```

Comment:
can you explain why we're using the major version of this dependency now?  I assume it was intentional but if not i always worry about automatic updates clashing with other dependencies down the road...

Reply:
As you said, we will delete `^`! Thanks👍